MODULE.bazel

```diff
@@ -0,0 +1,83 @@
+archive_override(
+    module_name = "rules_boost",
+    integrity = "sha256-PcGxb6Ffi1CajH1PIqlAJ6mgqQ5/XW3NNL7TLZvlSBs=",
+    strip_prefix = "rules_boost-64bf4814222a6782fd0e7536532a257d7fdc9d80",
+    urls = "https://github.com/nelhage/rules_boost/archive/64bf4814222a6782fd0e7536532a257d7fdc9d80.tar.gz",
+)
+
+non_module_boost_repositories = use_extension("@com_github_nelhage_rules_boost//:boost/repositories.bzl", "non_module_dependencies")
+use_repo(
+    non_module_boost_repositories,
+    "boost",
+)
+
+bazel_dep(name = "abseil-cpp", version = "20240116.1")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "crc32c", version = "1.1.0")
+bazel_dep(name = "fmt", version = "8.1.1")
+bazel_dep(name = "googletest", version = "1.14.0.bcr.1")
+bazel_dep(name = "liburing", version = "2.5")
+bazel_dep(name = "lz4", version = "1.9.4")
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "protobuf", version = "26.0")
+bazel_dep(name = "re2", version = "2023-09-01")
+bazel_dep(name = "rules_foreign_cc", version = "0.10.1")
+bazel_dep(name = "rules_proto", version = "6.0.2")
+bazel_dep(name = "yaml-cpp", version = "0.8.0")
+bazel_dep(name = "zlib", version = "1.3")
+bazel_dep(name = "zstd", version = "1.5.6")
+
+bazel_dep(name = "buildifier_prebuilt", version = "6.4.0", dev_dependency = True)
+bazel_dep(name = "rules_python", version = "0.33.2", dev_dependency = True)
+
+python = use_extension("@rules_python//python/extensions:python.bzl", "python", dev_dependency = True)
+python.toolchain(
+    ignore_root_user_error = True,
+    is_default = True,
+    python_version = "3.12",
+)
+
+pip = use_extension("@rules_python//python/extensions:pip.bzl", "pip")
+pip.parse(
+    extra_pip_args = ["--require-hashes"],
+    hub_name = "python_deps",
+    python_version = "3.12",
+    requirements_lock = "//bazel/thirdparty:requirements.txt",
+)
+use_repo(pip, "python_deps")
+
+non_module_dependencies = use_extension("//bazel:extensions.bzl", "non_module_dependencies")
+use_repo(non_module_dependencies, "ada")
+use_repo(non_module_dependencies, "avro")
+use_repo(non_module_dependencies, "base64")
+use_repo(non_module_dependencies, "c-ares")
+use_repo(non_module_dependencies, "hdrhistogram")
+use_repo(non_module_dependencies, "hwloc")
+use_repo(non_module_dependencies, "krb5")
+use_repo(non_module_dependencies, "libpciaccess")
+use_repo(non_module_dependencies, "libxml2")
+use_repo(non_module_dependencies, "lksctp")
+use_repo(non_module_dependencies, "numactl")
+use_repo(non_module_dependencies, "openssl")
+use_repo(non_module_dependencies, "openssl-fips")
+use_repo(non_module_dependencies, "rapidjson")
+use_repo(non_module_dependencies, "roaring")
+use_repo(non_module_dependencies, "seastar")
+use_repo(non_module_dependencies, "snappy")
+use_repo(non_module_dependencies, "unordered_dense")
+use_repo(non_module_dependencies, "wasmtime")
+use_repo(non_module_dependencies, "xxhash")
```

Comment:
Why not https://registry.bazel.build/modules/xxhash?

Reply:
See https://github.com/redpanda-data/redpanda/pull/20849#discussion_r1672600891